### DISCUSSION TITLE
Can't update ender 3 v2 screen firmware. Board V1.4

### DISCUSSION THREAD

**Discussion by fish7221** [Jyers/Marlin#1918] [Q&A] [answered]
Hello Friends,
I am new to 3d printing.
Just flashed latest Jyers firmware to mainboard (4.2.2), But i can't seem to flash the screen firmware.
I tried the DWIN_SET (Without identifier) copied to different sizes of formatted fat32 sd cards (4gb and 32gb formatted to FAT32), And also tried the DACAI Private folder, But both of them does not flash my display.
I would appreciate help.
Thank you

**Comment by BlueWings172** [comment, marked as answer]
@fish7221 

The SD card should also be formatted at 4096 bytes Allocation Unit Size. Try again with 4096 if you haven't use it before. If it doesn't work, check [this thread](https://github.com/Jyers/Marlin/discussions/1768).

**Reply by fish7221** [reply]
Thank you so much BlueWings172. The 4K format PLUS the DACAI firmware managed to be flashed successfully.
